mw-webapp/src/dataAccessLogic/WayPreviewDAL.ts

```diff
@@ -79,17 +79,20 @@ export class WayPreviewDAL {
   /**
    * Get User ways preview based of provided type
    */
-  public static async getUserWaysPreview(uuid: string, type: "OwnWays" | "MentoringWays"): Promise<WayPreview[]> {
+  public static async getUserWaysPreview(uuid: string, type: "Own" | "Mentoring" | "Favourite"): Promise<WayPreview[]> {
```

Comment:
"Favorite" is the right word. Please check the correct spelling of the word in all files

Reply:
[Favorite and Favourite are both correct](https://www.merriam-webster.com/grammar/favorite-or-favourite-usage)